helix-term/src/commands.rs

```diff
@@ -4327,7 +4327,6 @@ fn shrink_selection(cx: &mut Context) {
         // try to restore previous selection
         if let Some(prev_selection) = view.object_selections.pop() {
             if current_selection.contains(&prev_selection) {
-                // allow shrinking the selection only if current selection contains the previous object selection
```

Comment:
This is still true, so we should not remove this comment.

Reply:
Yes it still holds true, but the comment is reduntand an out of place. Usually, they explain the proceeding code. But here its explaining the if statement from within.  And it is not adding any context as the code is extremely self expanatory.

```rust
if current_selection.contains(&prev_selection) {
    doc.set_selection(view.id, prev_selection);
```

Reply:
The code is pretty self explanatory, but comments aren't solely to help explain what code is doing; they also help clarify what the original intention behind the code was. Anyone looking at this can see the logic is such that it will only accept a previous selection if the current one contains it, but it's not always clear *why*. Having a comment written in plain English makes it clear that this was in fact intentional.

If you take issue with placement, it would be fine to move it.